test/unittests/state_transition.cpp

```diff
@@ -0,0 +1,75 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "state_transition.hpp"
+
+namespace evmone::test
+{
+void state_transition::SetUp()
+{
+    pre.insert(tx.sender, {.nonce = 1, .balance = tx.gas_limit * tx.max_gas_price + tx.value + 1});
+    pre.insert(*tx.to, {.nonce = 1});
+
+    // Default expectations.
+    expect.post[Coinbase].exists = true;
+    expect.post[tx.sender].exists = true;
+    expect.post[*tx.to].exists = true;
```

Comment:
A bit annoying that to make creation transaction, we would need to unset `tx.to` but also not forget to remove it from `expect`. But maybe okay, as it's much less often than using default.

Reply:
Maybe a member function helper like `SetupCreationTransaction()` could help, but not sure.